duffel_api/models/airline.py

```diff
@@ -11,6 +11,8 @@ class Airline:
     id: str
     name: str
     iata_code: Optional[str]
+    logo_lockup_url: Optional[str]
```

Comment:
**future:** We need to add `conditions_of_carriage_url` [1] as well.

[1] https://duffel.com/docs/api/v1/airlines/schema#airlines-schema-conditions-of-carriage-url.